src/index.js

```diff
@@ -1,11 +1,26 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from './reducers/index';
 import './index.css';
-import App from './App';
+import App from './components/App';
+
+const initialState = {
+  books: [
+    { id: Math.floor(Math.random() * 30).toString(), title: 'Title 1', category: 'Kids' },
+    { id: Math.floor(Math.random() * 30).toString(), title: 'Title 2', category: 'Action' },
+    { id: Math.floor(Math.random() * 30).toString(), title: 'Title 3', category: 'Learning' },
+  ],
+};
+
+const store = createStore(rootReducer(initialState));
```

Comment:
The second [optional] argument to [`createStore`](https://redux.js.org/api/createstore) is the initial state. If you pass the `initialState` to `createStore` that way, you can simplify your `rootReducer` as it won't need to worry about setting that initial state:

```js
const store = createStore(rootReducer, initialState);
```